lib/pages/authentication-page.dart

```diff
@@ -187,7 +189,10 @@ class AuthenticationPageState extends State<AuthenticationPage> {
     _authenticationService.googleSignIn().then((User user) async {
       SharedPreferences prefs = await SharedPreferences.getInstance();
       await prefs.setString('mode', "google");
-      await _messagingService.setUserAppToken(user.email);
+      if (_preferencesService.getBool("onSharedWishlist") == null ||
```

Comment:
expression can be simplified is PreferencesService exposes a "enableNotifications" method